tests/ui/parallel-rustc/cycle_crash.rs

```diff
@@ -0,0 +1,6 @@
+//@ compile-flags: -Z threads=2
+//@ check-fail
```

Comment:
```suggestion
```

That's the default and shouldn't be necessary?

Reply:
removed